articles/introduction-to-wireless-networking/index.md

```diff
@@ -0,0 +1,122 @@
+### Introduction
+
+Wireless networks have exploded the world currently, all the same, we all hate wires all over, or don't we? Or perhaps not!  
+It has even replaced desktops that had connectors (RJ-45) on the wall.  
+Okay, let's say you go to a big hotel or airport, they will have a wireless network that you can use to access the internet.  
+The world has undergone a lot of internet evolution. 
+I would tell you this for sure if you wanted to buy a laptop that had the specification of a desktop with an RJ-45 connector on the wall,
+you would pay the double price.  
+Nowadays, the laptop price is affordable, or isn't it?  
+
+In this article, we are going to discuss the following:
+
+* Definition of wireless networking
+* How wireless network works.
+* Types of wireless networks
+* Wireless networking standards  
+* Benefits of wireless networking
+
+
+### A Quick look at networking
+
+Computer network refers to devices, such as computers, fax or even printers connected or linked to each other to communicate and share resources. It makes use of transmission media such as wires and cables.
+For example, in Local Area Networks (LAN), computers are interconnected in a small area such as an office via communication cables.  
+
+The main reason for connecting several devices on a network is to share resources. For example, an office has one printer shared, it can be connected to all computers within this office on a network to allow usage by each department.  
+
+You ought to have also learned that computer networks have five components.  
+They include  ***message, sender, receiver, protocol, and Transmission media***.  
+
+
+### What's wireless networking?
+
+Wireless networking refers to the technology where devices communicate without being tethered to any wires or cables.  
+As we have already learned, in wired networks, computers are connected physically via transmission media,  
+which is not the case with Wireless networks.  
+Now, wired networks have one drawback, that is, your desktop or laptop has to be physically connected or tethered   
+to a router or access switch, let's say via ethernet, minimizing the portability of your computers.  
+
+Obviously, in wireless, we don't use cables but radio waves instead.
+But do you remember **Carrier-sense multiple access with collision detection (CSMA/CD)** used in half-duplex networks?    
+Wireless networking is always half-duplex where the transmission can only take place in a single direction.   
+The main reason why wireless is half-duplex is because of sending and receiving on a single frequency.    
+Discussing this always leads to arguments on collisions, but that's beyond the scope of this article.  
+
+### How does a wireless network works?
+While learning about computer networks, you might have come across radio frequency technology.    
```

Comment:
Don't come across as expecting the reader to know everything in the article. Using phrases such as you might have come across, it is obvious, etc., sounds discouraging. Instead, use encouraging phrases. I have observed this throughout the article.